caddy/php-server.go

```diff
@@ -93,6 +94,10 @@ func cmdPHPServer(fs caddycmd.Flags) (int, error) {
 		} else if filepath.IsLocal(root) {
 			root = filepath.Join(frankenphp.EmbeddedAppPath, root)
 		}
+
+		if err := os.Setenv("PHP_INI_SCAN_DIR", ":"+frankenphp.EmbeddedAppPath); err != nil {
```

Comment:
I propose to change the env var only if:

* it isn't already set
* the embedded app contains a `php.ini` file

WDYT?

Reply:
What about appending the embedded path if the `PHP_INI_SCAN_DIR` already exists? I.E if the `PHP_INI_SCAN_DIR` env variable exists and the embedded path contains a php.ini, then we append the embedded path to the existing `PHP_INI_SCAN_DIR` value?

Reply:
This works for me, as long as we change the variable only if a `php.ini` file is present at the root of the app.